rules/app.bzl

```diff
@@ -136,7 +136,6 @@ def ios_application(
         deps = deps,
         frameworks = frameworks,
         families = families,
-        output_discriminator = None,
         infoplists = select(processed_infoplists),
         testonly = testonly,
         **application_kwargs
```

Comment:
Perhaps you could add  this to `application_kwargs` for 7

Reply:
Same issue if it's in `kwargs` or provided explicitly:
```
ERROR: /Users/mattrobinson/Developer/github/bazel-ios/rules_ios/tests/ios/unit-test/test-imports-app/BUILD.bazel:19:16: //tests/ios/unit-test/test-imports-app:TestImports-App: no such attribute 'output_discriminator' in 'ios_application' rule
ERROR: package contains errors: tests/ios/unit-test/test-imports-app
ERROR: /Users/mattrobinson/Developer/github/bazel-ios/rules_ios/tests/ios/vmd/test-app-host/BUILD.bazel:3:16: //tests/ios/vmd/test-app-host:TestAppHost: no such attribute 'output_discriminator' in 'ios_application' rule
ERROR: package contains errors: tests/ios/vmd/test-app-host
```